rules/build_sector.smk

```diff
@@ -1422,8 +1422,9 @@ rule prepare_sector_network:
             if config_provider("sector", "district_heating", "ates", "enable")(w)
             else []
         ),
-        hourly_fuel_costs=resources("hourly_fuel_costs_with_lignite.csv"),
+        hourly_fuel_costs="validation/ember_data/hourly_fuel_costs_with_lignite.csv",
         hourly_co2_prices="validation/ember_data/hourly_co2_prices_with_snapshots_2023.csv",
+        coal_data="validation/ember_data/coal_data.xlsx",
```

Comment:
can you save the corresponding data as `.csv` instead (if multiple tabs exist, we can create multiple `.csvs` instead of one one .xlsx with mutliple tabs)